mirror-maker-2/active-passive-ssl/config/connect-mirror-maker-template.properties

```diff
@@ -0,0 +1,93 @@
+
+#target and source configurations. Default value for the following settings is 3.
+#If you want more details about those internal configurations, please see https://docs.confluent.io/home/connect/userguide.html#kconnect-internal-topics
+#and https://docs.confluent.io/platform/current/connect/references/allconfigs.html#distributed-worker-configuration
+{{.Env.TARGET_NAME}}.config.storage.replication.factor=3
+{{.Env.SOURCE_NAME}}.config.storage.replication.factor=3
+#
+{{.Env.TARGET_NAME}}.offset.storage.replication.factor=3
+{{.Env.SOURCE_NAME}}.offset.storage.replication.factor=3
+#
+{{.Env.TARGET_NAME}}.status.storage.replication.factor=3
+{{.Env.SOURCE_NAME}}.status.storage.replication.factor=3
+
+# MirrorMaker configuration. Default value for the following settings is 3
+offset-syncs.topic.replication.factor=3
+heartbeats.topic.replication.factor=3
+checkpoints.topic.replication.factor=3
+
+tasks.max=10
+replication.factor=3
+refresh.topics.enabled=true
+refresh.topics.interval.seconds=30
+sync.group.offsets.interval.seconds=10
+emit.checkpoints.interval.seconds=10
+
+sync.topic.configs.enabled={{.Env.SYNC_TOPIC_CONFIGS}}
+sync.topic.acls.enabled={{.Env.SYNC_TOPIC_ACLS}}
+
+# Enable heartbeats and checkpoints
+{{.Env.SOURCE_NAME}}->{{.Env.TARGET_NAME}}.emit.checkpoints.enabled=true
+{{.Env.SOURCE_NAME}}->{{.Env.TARGET_NAME}}.sync.group.offsets.enabled = true
+{{.Env.SOURCE_NAME}}->{{.Env.TARGET_NAME}}.emit.heartbeats.enabled=true
+{{.Env.SOURCE_NAME}}->{{.Env.TARGET_NAME}}.sync.group.offsets.interval.seconds=10
+
+{{.Env.TARGET_NAME}}->{{.Env.SOURCE_NAME}}.emit.heartbeats.enabled=false
+{{.Env.TARGET_NAME}}->{{.Env.SOURCE_NAME}}.emit.checkpoints.enabled=false
+{{.Env.TARGET_NAME}}->{{.Env.SOURCE_NAME}}.sync.group.offsets.enabled = false
+
+# allow very large sizes for messages, as default is 1MB
+{{.Env.SOURCE_NAME}}.consumer.max.request.size=1572864000
+{{.Env.SOURCE_NAME}}.producer.max.request.size=1572864000
+
+{{.Env.TARGET_NAME}}.consumer.max.request.size=1572864000
+{{.Env.TARGET_NAME}}.producer.max.request.size=1572864000
+
+# SSL properties for connecting to SSL enabled source clusters
+{{ if eq .Env.SOURCE_SSL_ENABLE "true" }}
+{{.Env.SOURCE_NAME}}.ssl.enabled.protocols=TLSv1.2,TLSv1.1,TLSv1
+{{.Env.SOURCE_NAME}}.security.inter.broker.protocol=SSL
+{{.Env.SOURCE_NAME}}.security.protocol=SSL
+{{.Env.SOURCE_NAME}}.ssl.endpoint.identification.algorithm=
+{{.Env.SOURCE_NAME}}.ssl.truststore.location={{.Env.SOURCE_TRUSTSTORE}}
+{{.Env.SOURCE_NAME}}.ssl.truststore.password={{.Env.SOURCE_TRUSTSTORE_PASSWORD}}
+{{.Env.SOURCE_NAME}}.ssl.keystore.location={{.Env.SOURCE_KEYSTORE}}
+{{.Env.SOURCE_NAME}}.ssl.keystore.password={{.Env.SOURCE_KEYSTORE_PASSWORD}}
+{{ end }}
+
+{{ if eq .Env.TARGET_SSL_ENABLE "true" }}
+# SSL properties for connecting to SSL enabled target clusters
+{{.Env.TARGET_NAME}}.security.inter.broker.protocol=SSL
+{{.Env.TARGET_NAME}}.ssl.enabled.protocols=TLSv1.2,TLSv1.1,TLSv1
```

Comment:
is anything stopping us from only supporting TLSv1.2 (since the others are [deprecated](https://datatracker.ietf.org/doc/html/rfc8996))?